Running-Mastodon/Nanobox-Guide.md

```diff
@@ -11,29 +11,39 @@ Development
 
 You will need Nanobox installed, along with Docker if you're on Linux (Windows
 and macOS can use Docker Native, but the bundled VirtualBox is more performant
-while the Docker team works out some filesystem speed issues). The process is
-simple - clone the repo, set a few variables with `nanobox evar add local
-{VARIABLE}={value}` (see below on which ones need to be set), and run `nanobox
-run` to get to a console. It will take some time to build your local dev
-environment, but once it's done, simply set up the DB using `bundle exec rake
-db:setup` as normal, and you're off.
+while the Docker team works out some filesystem speed issues). Once Nanobox is
+installed, the process is simple - clone the Mastodon repo, set a few variables
+with `nanobox evar add local {VARIABLE}={value}` (see below on which ones need
+to be set), and run `nanobox run` to get to a console. It will take some time to
+build your local dev environment, but once it's done, simply set up the DB using
+`bundle exec rake db:setup` as normal, and you're off.
 
 Production
 ----------
 
-To deploy, you'll need to create an application in [your Nanobox dashboard](https://dashboard.nanobox.io/apps)
-(which requires a [Nanobox.io](https://dashboard.nanobox.io/users/register)
-account), clone the repo (if you haven't already set up the local development
-environment), set the new app as your deploy target with `nanobox remote add
-{app-name}`, set up the variables below using either `nanobox evar add
-{VARIABLE}={value}` or the app's dashboard, and then run `nanobox deploy`.
+To deploy, you'll need to create an application in [your Nanobox
+dashboard](https://dashboard.nanobox.io/apps) (which requires a _free_
+[Nanobox.io](https://dashboard.nanobox.io/users/register) account), clone the
+Mastodon repo (if you haven't already set up the local development environment),
+set the new app as your deploy target with `nanobox remote add {app-name}`, set
+up the variables below using either `nanobox evar add {VARIABLE}={value}` or the
+app's dashboard, and then run `nanobox deploy`. It will take a while to build
+everything for deployment the first time, but the process handles itself.
 
 Updating
 --------
 
-To update, simply grab the latest tagged version with `git fetch && git checkout
-$(git tag | tail -n 1)`, then re-run `nanobox deploy` - Nanobox automatically
-handles the rest.
+To update in production, simply grab the latest tagged version with `git fetch &&
+git checkout $(git tag | tail -n 1)`, then re-run `nanobox deploy` - Nanobox
```

Comment:
Looks good! we recently updated the production guide with a more robust command for checking out the latest version though, instead of just hoping the most recent tag is stable. Can you use `git checkout $(git tag -l | grep -v 'rc[0-9]*$' | sort -V | tail -n 1)` instead?

Looks good other then that!

Reply:
Not sure how I missed that, since I checked the prod guide to ensure I hadn't missed anything _else_ in the last few releases. :grinning: I'd always meant to do something about stripping out the RC versions...

Reply:
Fixed, by the way.